packages/instantsearch-react/src/impl/Range.js

```diff
@@ -0,0 +1,50 @@
+import React, {PropTypes, Component} from 'react';
+import Slider from 'rc-slider';
+
+// @TODO: rc-slider doesn't support our customization options relative to
+// theming. We'll need to implement our own slider down the road.
+export default class Range extends Component {
+  static propTypes = {
+    min: PropTypes.number.isRequired,
+    max: PropTypes.number.isRequired,
+    value: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
Ok so it's the values of the range? The actual values, not the boundaries (which are min and max)?